python/interpret_community/explanation/explanation.py

```diff
@@ -1215,15 +1225,15 @@ def _convert_to_list(self, data):
 
         :param data: The data to be converted.
         :type data: np.array, pd.DataFrame, list, scipy.sparse
-        :return: The data converted to a list (except for sparse which is unchanged).
-        :rtype: list | scipy.sparse | list[scipy.sparse]
+        :return: The data converted to a list (except for sparse or string which is unchanged).
```

Comment:
Might want to clarify that string in this case means Dataset reference?

Reply:
done!  changed to dataset ID